root/report/components/ArtistList.js

```diff
@@ -11,28 +11,33 @@ import * as React from 'react';
 import type {ColumnOptionsNoValue} from 'react-table';
 
 import PaginatedResults from '../../components/PaginatedResults.js';
-import Table from '../../components/Table.js';
+import useTable from '../../hooks/useTable.js';
 import {
   defineEntityColumn,
   defineTextColumn,
 } from '../../utility/tableColumns.js';
 
-type Props<D: {+artist: ?ArtistT, ...}> = {
+type NullableArtistResultT = {+artist: ?ArtistT, ...};
```

Comment:
Why separate this here, but not elsewhere? (Release, etc?). It seems sensible to me but probably better to do it consistently.

Reply:
I think I started with this file first, and wanted to create separate nullable and non-nullable types to account for the fact that `existingArtistItems` filters out the null ones already, and then change the `columns` definitions so that they don't have to unnecessarily check `result.artist` for null everywhere. Then I realized it was a lot of work and that we should probably just filter out undef items on the server. So I reverted most of the changes and probably forgot this one.

Reply:
Feel free to change this back if you want then :) (and add a ticket for the filter, possibly?)